en/docs/using-wso2-identity-server/password-recovery.md

```diff
@@ -38,21 +33,25 @@ notifications. The flow of this method is as follows:
 Follow the steps given below to set up and try out password recovery
 with email notification.
 
-!!! tip
-    
-    Before you begin
+!!! tip "Before you begin"
     
     Ensure that the " `         IdentityMgtEventListener        ` " with the
     `         orderId=50        ` is set to **false** and that the Identity
```

Comment:
the "orderId" parameter should change to "priority" and the toml configs should be added to the file